### PR TITLE
release 8.5.0 (and 8.5.1)

### PR DESCRIPTION
**Pull request by Klowner**
Just updating the numbers and building a new image appears to be working fine for me so far. I saw you're traveling, so hopefully this will make your life a little easier. :)

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 This file only reflects the changes that are made in this image. Please refer to the upstream GitLab [CHANGELOG](https://gitlab.com/gitlab-org/gitlab-ce/blob/master/CHANGELOG) for the list of changes in GitLab.
 
+**8.5.0**
+- gitlab: upgrade to CE v8.5.5
```
Comment (alexmchale) [inline Changelog.md]: Typo here :)

**Comment by Klowner** [comment]
Doh, thank you @alexmchale. fixed

**Comment by Klowner** [comment]
quay.io/klowner/gitlab:8.5.0 if anyone wishes to give it a whirl

**Comment by ninja-** [comment]
woah I was about to send a pr like this ;)

**Comment by ninja-** [comment]
btw make it 8.5.1 already

**Comment by Klowner** [comment]
just for you, @ninja- :dancers:

**Comment by sameersbn** [comment]
Thanks for your efforts. I have made the 8.5.0 release. 8.5.1 coming up...

**Pull request closed without merging**